### PR TITLE
fix: back button  now working on collection quest

### PR DESCRIPTION
**Pull request by martinvibes**
# Pull Request type

Please add the labels corresponding to the type of changes your PR introduces:

- Bugfix
- Feature
- Code style update (formatting, renaming)
- Refactoring (no functional changes, no API changes)
- Build-related changes
- Documentation content changes
- Testing
- Other (please describe):

Resolves: #NA

## Summary by CodeRabbit

- **New Features**
	- Enhanced navigation in the Category component, allowing users to return to a specific tab based on previous interactions.
	- Added persistent tab management in the Quest and Collection Tabs, enabling users to maintain their active tab across sessions.

- **Usability Improvements**
	- Improved user experience by remembering the last active tab when users return to the component.

### CODE REVIEW

**Comment by martinvibes** [comment]
hello @fricoben kindly review

File: app/categories/[category]/category.tsx [hunk 1]
```diff
@@ -32,6 +32,17 @@ const Category: FunctionComponent<CategoryPageProps> = ({ categoryName }) => {
...
+  const tabRoutes = ["/", "/", "/"];
+
+  const handleBack = () => {
+    const activeTab = localStorage.getItem("activeTab");
+    if (activeTab) {
+      router.push(tabRoutes[parseInt(activeTab, 10)]);
+    } else {
+      router.back();
+    }
+  };
```
[the remark this answers is not in the document]
Comment (fricoben) [reply app/categories/[category]/category.tsx:44]: Agree, Let's implement it !

**Review by fricoben — CHANGES_REQUESTED** [review]
I can not test again cause it doesn't build. 

Please use `npm run build` and make sure it builds before sending a PR

**Comment by fricoben** [comment]
@martinvibes please fix or I'll need to reassign this issue to someone else !

**Comment by martinvibes** [comment]
@fricoben sorry for the delay i'm working on it

**Pull request closed without merging**